README.rst

```diff
@@ -28,16 +29,14 @@ Features
 
 
 Usage
------
-The most straightforward way is to use the :func:`dateparser.parse` function,
-that wraps around most of the functionality in the module.
+=====
 
-.. automodule:: dateparser
-   :members: parse
+The most straightforward way is to use the `dateparser.parse` function,
```

Comment:
Yeah, this PR doesn't have any issues with `restview`, though It would be cool if we find a way to not use `:func:` in the `README.rst` (so pypi won't break markdown) but still keep it for `docs/` so it would appear in `readthedocs`